src/testRunner.js

```diff
@@ -0,0 +1,16 @@
+const http = require('http')
+
+const { server } = require('./config')
+const logger = require('./lib/logger')
+const app = require('./web/server')
+require('./tests/mocks/service_transactions_statistics_mock.js')
```

Comment:
We will need to think through a way to set up the correct mocks on a per test basis. This is generically run at the top level but will only work for this test.

I wonder if there's a way to use a Cypress 'task' to set up the mocks https://docs.cypress.io/api/commands/task.html

Reply:
I tested this with another test and put the mocks in a separate file and after running it it 'hit' the mocks in sequential order. I'm not sure how we can use the Cypress task for setting up the mocks as they have to be setup before the server is set up and only after can Cypress run if I'm not mistaken?